packages/wonder-blocks-tabs/src/components/tabs-dropdown.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from "react";
+import {ActionItem, ActionMenu} from "@khanacademy/wonder-blocks-dropdown";
+import Button from "@khanacademy/wonder-blocks-button";
+import caretDown from "@phosphor-icons/core/bold/caret-down-bold.svg";
+import {StyleSheet} from "aphrodite";
+import {border, semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+import checkCircleIcon from "@phosphor-icons/core/fill/check-circle-fill.svg";
+import {PhosphorIcon} from "@khanacademy/wonder-blocks-icon";
+import {addStyle, StyleType} from "@khanacademy/wonder-blocks-core";
+
+const StyledDiv = addStyle("div");
+
+type TabDropdownItem = {
+    /**
+     * The label of the tab
+     */
+    label: string;
```

Comment:
`ActionItem` only supports strings for the `label` prop, we aren't able to provide the same flexibility as `Tabs` where it accepts any React element or a render function. 

Because of this, the `ResponsiveTabs` component will support only a subset of the flexibility provided by `Tabs`. We could solve for this in other ways, but I think for now we can standardize and limit the API so it works well with both Tabs and TabsDropdown. More to come in upcoming PRs!

Reply:
`ActionItem` supports flexible labels now. https://github.com/Khan/wonder-blocks/blob/6d351755817825e0c3d66e81941e1fe775ec610d/packages/wonder-blocks-dropdown/src/components/action-item.tsx#L20

This aligns with how option items work as well. Maybe this can be considered for this API as well?